src/Sulu/Bundle/HttpCacheBundle/DependencyInjection/SuluHttpCacheExtension.php

```diff
@@ -14,17 +14,32 @@
 use Symfony\Component\Config\Definition\Exception\InvalidConfigurationException;
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
 use Symfony\Component\DependencyInjection\Loader;
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
 
-class SuluHttpCacheExtension extends Extension
+/**
+ * Container extension for sulu-http-cache bundle.
+ */
+class SuluHttpCacheExtension extends Extension implements PrependExtensionInterface
 {
     /**
      * {@inheritdoc}
      */
-    public function getConfiguration(array $config, ContainerBuilder $container)
+    public function prepend(ContainerBuilder $container)
     {
-        return new Configuration();
+        if ('prod' === $container->getParameter('kernel.environment')) {
```

Comment:
However, the way you have implemented it now caching is only deactivated for the `prod` environment 😉